flutter_web_auth_2/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -36,12 +36,18 @@
             android:exported="true">
             <intent-filter android:label="flutter_web_auth_2">
                 <action android:name="android.intent.action.VIEW" />
+
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
+
                 <data android:scheme="foobar" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.SEND" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:mimeType="text/plain" />
+            </intent-filter>
```

Comment:
I see that this is needed if `autoVerify="true"`. But is it really needed in the example?

Reply:
the scheme is not http/https, so don't need add` autoVerify="true"`.